tdms/src/iterator.cpp

```diff
@@ -6368,27 +6347,6 @@ if(runmode == rm_complete && (nvertices>0) )
   free(K1);
   free(dims);
   free(label_dims);
-  if(is_disp || is_disp_ml){
-    destroy_auxilliary_mem(I_tot, J_tot, K_tot,
-                           &E_nm1.xy, &E_nm1.xz,
-                           &E_nm1.yx, &E_nm1.yz,
-                           &E_nm1.zx, &E_nm1.zy,
-                           &J_s.xy, &J_s.xz,
-                           &J_s.yx, &J_s.yz,
-                           &J_s.zx, &J_s.zy,
-                           &J_nm1.xy, &J_nm1.xz,
-                           &J_nm1.yx, &J_nm1.yz,
-                           &J_nm1.zx, &J_nm1.zy);
-
-  }
-
-  //fprintf(stderr,"Pos 22\n");
-  if(is_cond){
-    destroy_auxilliary_mem_conductive(I_tot, J_tot, K_tot,

```

Comment:
Are these destroys now taken care of elsewhere?

Reply:
yep – should be taken care of in the e.g. E_nm destructor